src/links.md

```diff
@@ -98,6 +98,8 @@ Keep lines sorted.
 [num]: https://docs.rs/num/
 [num_cpus-badge]: https://badge-cache.kominick.com/crates/v/num_cpus.svg?label=num_cpus
 [num_cpus]: https://docs.rs/num_cpus/
+[postgres-badge]: https://badge-cache.kominick.com/crates/v/postgres.svg?label=postgres
+[postgres]: https://crates.io/crates/postgres/
```

Comment:
This should link to [https://docs.rs/postgres/](https://docs.rs/postgres/) instead.

Reply:
Fixed.